packages/documentation/src/stories/foundations/icons/icon.styles.scss

```diff
@@ -28,13 +28,13 @@ $resizer-max-width: 64px;
 }
 
 .my-1022-icon {
-  @include post.post-icon($name: '1022', $base: '/post-icons');
+  @include post.post-icon($name: '1022');
 }
 
 .my-accessibility-icon {
-  @include post.post-icon($name: 'accessibility', $base: '/post-icons');
+  @include post.post-icon($name: 'accessibility');
 }
 
 .my-handvictory-icon {
-  @include post.post-icon($name: 'handvictory', $base: '/post-icons', $color: #666666);
+  @include post.post-icon($name: 'handvictory', $color: #666666);
 }
```

Comment:
Where is that even used?
If it's not used in any example, you could remove the styles.